container/webui/run_openqa.sh

```diff
@@ -39,6 +39,8 @@ function all_together_apache() {
   su geekotest -c /usr/share/openqa/script/openqa-webui-daemon
 }
 
+usermod --shell /bin/sh geekotest
```

Comment:
I don't understand. If the user "geekotest" does not exist, how does this command help?

Reply:
During the tests I did geekotest exists but doesn't have a console assigned
`
webui_db_init_1  | geekotest:x:479:479:openQA user:/dev/null:/sbin/nologin
`

Reply:
This generates the problem: "This account is currently not available."

Reply:
hm, ok. Please phrase your git commit message subject line to say what your change fixes, describing the reason for the change, not the "what changed"

Reply:
Done

Reply:
I guess a commit message like "containers: Fix use of `su` by providing shell for geekotest". Also note that you're changing the *shell* here, not the console.

Reply:
Changed